pkgs/top-level/all-packages.nix

```
@@ -35751,6 +35751,8 @@ with pkgs;
 
   with-shell = callPackage ../applications/misc/with-shell { };
 
+  wizer = callPackage ../development/tools/wizer { };
```

This will fix the `x86_64-darwin` build.

```suggestion
  wizer = darwin.apple_sdk_11_0.callPackage ../development/tools/wizer { };
```

---

Looks like you forgot to commit this.

---

Ah, fun, GitHub isn't letting me commit this for whatever reason. ("This diff has recently been updated. Refresh and try again." Refreshing does not work! Thank you GitHub.)

Do you mind committing this + squashing the changes? Thanks!